ci/gpu/build.sh

```diff
@@ -65,11 +65,8 @@ if [ "$py_ver" == "3.6" ];then
 fi
 
 conda install "rmm=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
-              "dask>=2.15.0" "distributed>=2.15.0" "numpy>=1.16" "double-conversion" \
-              "rapidjson" "flatbuffers" "boost-cpp=1.72.0" "fsspec>=0.6.0" "dlpack" \
-              "feather-format" "cupy>=6.6.0,<8.0.0a0,!=7.1.0" "arrow-cpp=0.17.1" "arrow-cpp-proc=*=cuda" "pyarrow=0.17.1" \
-              "fastavro>=0.22.0" "pandas>=0.25,<0.26" "hypothesis" "s3fs" "gcsfs" \
-              "boto3" "moto" "httpretty" "streamz" "ipython=7.3*" "jupyterlab" "librdkafka>=1.3"
+              "rapids-build-env=$MINOR_VERSION.*" \
+              "rapids-notebook-env=$MINOR_VERSION.*"
```

Comment:
We should add the following lines below this, commented out. Maybe a short comment to explain what these lines are used for and a possible link to the recent docs?

```
conda remove -f rapids-build-env rapids-notebook-env
conda install "your-pkg=1.0.0"
```